learningresources/tests/test_models.py

```diff
@@ -145,7 +145,7 @@ def test_static_asset_filename_length(self):
                 FILE_PATH_MAX_LENGTH -
                 # minus the length of the base path of the Django storage and
                 # the base path of the file location temporary dir
-                len(os.path.join(base_path, temp_dir_path)) -
+                len('/'.join([base_path, temp_dir_path])) -
```

Comment:
it was giving a problem in that context: for some reason it was ignoring the first variable `base_path`, maybe because it does not start with a `/`, but it does not make much sense.
Maybe I need to investigate more, but considering that it is in a test and just to calculate a length, I think it does not matter